### ISSUE TITLE
 Manifest merger failed 

### ISSUE THREAD

**Issue by AntonioCuevaUrraco** [NativeScript/push-plugin#16]
Android side.

If the main manifest have a versionCode and/or versionName different that the push manifest the build fails. 
```
> Manifest merger failed with multiple errors, see logs
Command /Users/antoniocueva/SmarpShare/platforms/android/gradlew failed with exit code 1
```
And you have to manually change the numbers to match...Is there something we can do about this? what is the reason for the push manifest have pre set this values?

**Comment by AntonDobrev** [comment]
@AntonioCuevaUrraco 

Thanks for bringing the issue to our attention. 

This is a valid point, in fact you can safely remove the versionName and the versionCode from the plugin AmdroidManifest:

<manifest xmlns:android="http://schemas.android.com/apk/res/android">

**Comment by AntonDobrev** [comment]
Fixed in [0.0.15](https://github.com/NativeScript/push-plugin/commit/b16bf4c73887e9e05266792246a383533ff0060a)
